src/contradictory_claims/claims/utils.py

```diff
@@ -0,0 +1,276 @@
+from typing import Dict, Optional, List, Any
+
+import json
+import pandas as pd
+import numpy as np
+from overrides import overrides
+import torch
+from torch.nn.modules.linear import Linear
+import torch.nn.functional as F
+import os
+
+from allennlp.common import Params
+from allennlp.common.checks import check_dimensions_match
+from allennlp.data import Vocabulary
+from allennlp.modules import Seq2VecEncoder, TimeDistributed, TextFieldEmbedder, ConditionalRandomField, FeedForward
+from allennlp.modules.conditional_random_field import allowed_transitions
+from allennlp.models.model import Model
+from allennlp.nn import InitializerApplicator, RegularizerApplicator
+from allennlp.nn import util
+from allennlp.training.metrics import SpanBasedF1Measure, CategoricalAccuracy
+
+from allennlp.common.file_utils import cached_path
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.fields import Field, TextField, LabelField, SequenceLabelField, ListField
+from allennlp.data import Instance
+from allennlp.data.token_indexers import SingleIdTokenIndexer, TokenIndexer
+from allennlp.data.tokenizers import Token, Tokenizer, WordTokenizer
+from allennlp.predictors import Predictor
+
+from allennlp.common.util import JsonDict
+
+import spacy
+
+nlp = spacy.load('en_core_web_sm')
```

Comment:
Should we be using scispacy?

Reply:
It is only being used to break paragraphs => sentences, which are later broken into individual words